cpp/ql/src/Likely Bugs/Memory Management/ReturnStackAllocatedMemory.ql

```diff
@@ -75,14 +75,23 @@ class ReturnStackAllocatedMemoryConfig extends MustFlowConfiguration {
   }
 }
 
+predicate instrHasEnclosingCallable(VariableAddressInstruction var, Function f) {
```

Comment:
As this has little to do with callable. I'd suggest:
```suggestion
predicate instrHasEnclosingFunction(VariableAddressInstruction var, Function f) {
```

Reply:
Fixed in 7c411b4bad116c1e327baee3bee4ac69d92c58cd.